app/utils/get_filtered_records.py

```diff
@@ -144,33 +144,22 @@ def get_filtered_records(research_fields=False, filters=None, columns=None,
     if query_dicts is None or len(query_dicts) == 0:
         return []
 
-    result = []
-
     # Return all records if no filters are passed in
+    result = query_dicts
+
     if filters:
```

Comment:
@simonarocco do you think it would be cleaner to define all this filtering logic in SQL instead of python? 

I think we should definitely move date filtering to SQLAlchemy cause it's pretty simple, but not too sure about this.

Reply:
I'm not a fan of this python filtering logic at all, it's hard to read and seems over engineered when there is a better way to do filtering. I agree we should do filtering logic in SQL if we can. I can help @rahularoradfs turn this into sqlalchemy